docs/language/overview.md

```diff
@@ -1572,13 +1548,13 @@ the "in" operator, e.g.,
 
 ##### 7.2.1.6 Predicate Search Term
 
-Any Boolean-valued [function](functions/README.md) like `is()`, `has()`,
-`grep()` etc. and any [comparison expression](#comparisons)
+Any Boolean-valued [function](functions/README.md) like `is`, `has`,
+`grep` etc. and any [comparison expression](#62-comparisons)
```

Comment:
In other language docs it is indeed common to use the `()` as shorthand for describing a function. However, since in so many places in the doc the word "function" was already being used, I changed these to adopt that convention.